apps/datahub/src/app/home/search/organization-preview-datahub/organization-preview-datahub.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="w-72">
+  <div
+    class="border p-3 rounded-lg my-2 w-full h-48 flex justify-center items-center my-3"
+  >
+    <img [src]="logo" alt="title" />
```

Comment:
Should not work does it ?
```suggestion
    <img [src]="logo" [alt]="title" />
```